### DISCUSSION TITLE
Github is sharing the repos that I starred publicly.

### DISCUSSION THREAD

**Discussion by celikmustafa89** [community/community#35991] [Other Feature Feedback, Questions, & Ideas]
Github is showing the repositories that I have starred. I do not want to share the starred repos publicly. I could not find any configuration to close it. it is me, or there is no such conf? If there is no conf about that, can you add it :)

**Comment by utkarsh006** [comment]
- You cannot hide what you've starred on GitHub. Once it's starred, it's out there.


But try out this once : 

- Go to Settings > Public Profile > Contributions & Activity section.
- Check the box for Make profile private and hide activity .
- Click on Update Preferences button.

**Reply by celikmustafa89** [reply]
It hides all the information about my profile, achievements, overview projects, followers etc.
I just want to hide the starred repos. I think It is not possible yet.

**Comment by celikmustafa89** [comment]
Another suggestion is it possible to show the visit count of the repos.
for example, there is a repo called X-project. 
GitHub counts me for each visit to this project, and shows me the results on the page.
So, I can be able to know that I have been this repo before.
May be it can share details about the visiting dates, and the pages that I navigated

**Reply by jfullstackdev** [reply]
when you own the repo, you have it at the Insight, like mine 

![image](https://user-images.githubusercontent.com/47092464/204711304-70d10a32-7bf4-4f79-841f-4e7570b7fc84.png)

if you want to display it on the README, you have an external service for that, like what I'm using

![image](https://user-images.githubusercontent.com/47092464/204711401-3b2a6243-3410-4d6d-be4d-db14271d3f95.png)

check this out 

https://github.com/jdevstatic/page-metrics-tutorial

**Comment by jfullstackdev** [comment]
put it this way, when you like a Public Post, anyone can see it,

on GitHub, the same rules apply, so if you star a public repo, then that will always be public

when you star your own private repo, that will remain private or any other private repo you have access, it will not be public

**Reply by WEI567** [reply]
fLOHKzqAAAAAAElFTkSuQmCC

**Reply by WEI567** [reply]
fLOHKzqAAAAAAElFTkSuQmCC

**Reply by WEI567** [reply]
fLOHKzqAAAAAAElFTkSuQmCC

**Reply by WEI567** [reply]
fLOHKzqAAAAAAElFTkSuQmCC

**Comment by mohamedzhioua** [comment]
You cannot hide what you've starred on GitHub. Once it's starred, it's out there.
 But  GitHub has a beta feature which "hides your contributions and activity from your GitHub profile and from social features like followers, stars, feeds, leaderboards and releases."

To activate it:

Go to Settings > Public Profile > Contributions & Activity section
Check the box for Make profile private and hide activity.
Click on Update Preferences button.
![Re5Wh](https://user-images.githubusercontent.com/107249637/205464776-5217bb57-84ee-4d78-b26c-702f8218b160.png)

After activation, all your activity will be private .
really i hope github find a solution for your Question that allow us to hide all the repositories we starred

**Reply by candaceblewett42** [reply]
![image](https://user-images.githubusercontent.com/82898064/205468602-3ee5165a-a492-4736-a4bc-4f561bbbfce3.png)

**Reply by candaceblewett42** [reply]
![image](https://user-images.githubusercontent.com/82898064/205468616-91c5eca8-2ede-4603-a2af-2ca611f2d518.png)

**Reply by candaceblewett42** [reply]
Hey James Simpson so when I email you asking why you're in my data everywhere it's going to get hub why is that and why in the hell are you all over in my phone what is your problem are you obsessed with me do you need to get a life do you wish you were me do you wish you can wear me why don't you figure something else out to do with your time OK I know you wish you looked like me or your girlfriend looks like me but it's never gonna happen OK so get a life find something else to do with your time figure it out

**Reply by candaceblewett42** [reply]
- [ ] - ![image](https://user-images.githubusercontent.com/82898064/205468672-aa64893b-b0db-48c8-9b5a-617f1041d05f.png)

**Reply by atlslscsrv-app** [reply]
This may actually be a mix-up and I pray 🙏 it is , Hi I am a woman who has lost her man in 2020
So my question is, is that how long you have been receiving those random messages for Julia Frost? And if so that means...We need to talk about it and I will explain. If not apologize for interrupting your day and I will hopefully find out all of his ghost accounts B4 i die.😢🤗

**Comment by pataquets** [comment]
See #8626

**Comment by Santiagosaiz** [comment]
3d3266b34270122fac29886cae88dea120c51336

**Reply by Santiagosaiz** [reply]
_****_santiagosaiz4

**Comment by deepgithubforever** [comment]
Realtors

**Comment by Jonhenry786** [comment]
If GitHub is sharing your starred repos publicly, you can adjust your profile settings to keep them private. I had a similar issue while organizing projects with a [WhatsApp tool](https://lemcrm.co/), and changing the visibility settings fixed it!

**Comment by WEI567** [comment]
> Github is showing the repositories that I have starred. I do not want to share the starred repos publicly. I could not find any configuration to close it. it is me, or there is no such conf? If there is no conf about that, can you add it :)

**Comment by WEI567** [comment]
> Github is showing the repositories that I have starred. I do not want to share the starred repos publicly. I could not find any configuration to close it. it is me, or there is no such conf? If there is no conf about that, can you add it :)
